nix/mobile/default.nix

```diff
@@ -9,19 +9,52 @@ let
     version = "10.2.1";
   };
   xcodeWrapper = composeXcodeWrapper xcodewrapperArgs;
-  androidPlatform = callPackage ./android.nix { inherit config gradle; };
-  selectedSources =
-    [ status-go ] ++
-    lib.optional platform.targetAndroid androidPlatform;
+  androidPlatform = callPackage ./android { inherit config pkgs target-os mkFilter nodejs maven localMavenRepoBuilder projectNodePackage prod-build-fn; status-go = status-go.android; };
+  iosPlatform = callPackage ./ios { inherit config pkgs mkFilter xcodeWrapper projectNodePackage; status-go = status-go.ios; };
+  fastlane = callPackage ./fastlane { inherit stdenv target-os; };
+  selectedSources = [
```

Comment:
Can you explain this variable to me?

Reply:
I was hoping the naming/usage would be sufficient :-) There are quite a few expressions from which we need to gather buildInputs and shellHooks. Initially we were doing that manually, but it made more sense to do something like `lib.catAttrs "buildInputs" selectedSources` to avoid code repetition between sources for buildInputs and shellHooks.

Reply:
Okay.